### PR TITLE
General Updates

### PR DESCRIPTION
**Pull request by ASolom1**
Updates to address policies that control federation, comprehensively include federation with unmanaged Teams users and remove Skype's "preview" designation.

Note: all list structures need review + restructure to be easier to read.

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Teams/manage-external-access.md [hunk 1]
```diff
@@ -25,42 +25,61 @@ ms.localizationpriority: high
...
 
 - You want anyone else in the world who uses Teams to be able to find and contact you, using your email address. 
 
 ## Plan for external access
+External access policies include controls for each type of federation at both the tenant- and user-levels. Turning a policy off at the tenant-level turns it off for all users, regardless of their user-level setting. All external access settings are enabled by default.
 
-By default, external access is turned on in Teams, which means that your organization can communicate with all external domains. If you add blocked domains, all other domains will be allowed; and if you add allowed domains, all other domains will be blocked. The exception to this rule is if anonymous participants are allowed in meetings. There are three scenarios for setting up external access in the Teams admin center (**Users** > **External access**):
+Tenant-level settings can be changed via Teams Admin Center or PowerShell (Set-CSTenantFederationConfiguration). User-level settings can be changed via PowerShell (Set-CsExternalAccessPolicy). In later sections we’ve provided instructions on how to use Teams Admin Center and write PowerShell commands.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:43]:
```suggestion
Tenant-level settings can be changed via Teams admin center or PowerShell (Set-CSTenantFederationConfiguration). User-level settings can be changed via PowerShell (Set-CsExternalAccessPolicy). In later sections, we’ve provided instructions on how to use Teams admin center and write PowerShell commands.
```

File: Teams/manage-external-access.md [hunk 2]
```diff
@@ -25,42 +25,61 @@ ms.localizationpriority: high
...
+For reference, below is a description of the PowerShell controls.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:44]:
```suggestion
For reference, the following is a description of the PowerShell controls.
```

File: Teams/manage-external-access.md [hunk 3]
```diff
@@ -92,75 +111,67 @@ To block specific domains
 
 6. Click **Save**.
 
-Make sure the admin in the other Teams organization completes these same steps. For example, in their **allowed domains** list, their admin needs to enter the domain for your business if they limit the organizations that can communicate with their users.
+To communicate with another tenant, they must either enable Allow all external domains or add your tenant to their list of allowed domains by following the same steps above.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:114]:
```suggestion
To communicate with another tenant, they must either enable **Allow all external domains** or add your tenant to their list of allowed domains by following the same steps above.  
```

File: Teams/manage-external-access.md [hunk 4]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
+To communicate with another tenant, they must either enable Allow all external domains or add your tenant to their list of allowed domains by following the same steps above.  
 
-## Communicate with Skype users (preview)
+## Manage contact with external Teams users not managed by an organization
 
-Follow these steps to let Teams users in your organization chat with and call Skype users. Teams users can then search for and start a one-on-one text-only conversation or an audio/video call with Skype users and vice versa.
+Admins can choose to enable or disable communications with external Teams users that are not managed by an organization (“unmanaged”). If enabled, they can also further control if external users with unmanaged Teams accounts can initiate contact (see image below). If external user initiation is turned off, unmanaged Teams users will not be able to search the full email address to find organization contacts and all communications with unmanaged Teams users must be initiated by organization users.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:118]:
```suggestion
Admins can choose to enable or disable communications with external Teams users that are not managed by an organization (“unmanaged”). If enabled, they can also further control if external users with unmanaged Teams accounts can initiate contact (see the following image). If external user initiation is turned off, unmanaged Teams users will not be able to search the full email address to find organization contacts and all communications with unmanaged Teams users must be initiated by organization users.
```

File: Teams/manage-external-access.md [hunk 5]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
 
-  **Using the Microsoft Teams admin center**
+![image](https://user-images.githubusercontent.com/98335128/155242115-cdd65afe-62c4-40e5-8603-c1ac770103c0.png)
 
-1. In the left navigation, go to **Users** > **External access**.
+**Using the Microsoft Teams admin center**
 
-2. Turn on the **Allow users in my organization to communicate with Skype users** setting.
+1.	In the left navigation, go to Users > External access.<br>
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:124]:
```suggestion
1.	In the left navigation, go to **Users** > **External access**.<br>
```

File: Teams/manage-external-access.md [hunk 6]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
+               2.	Clear the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>
+      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
+               1.	Turn on the People in my organization can communicate with Teams users whose accounts aren't managed by an organization setting <br>
+               2.	Clear the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>
+      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>
+               1.	Turn on the People in my organization can communicate with Teams users whose accounts aren't managed by an organization setting<br>
+               2.	Select the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:134]:
```suggestion
               1.	Turn on the _People in my organization can communicate with Teams users whose accounts aren't managed by an organization_ setting.<br>
               2.	Select the _External users with Teams accounts not managed by an organization can contact users in my organization_ checkbox.<br>
```

File: Teams/manage-external-access.md [hunk 7]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
-4. Another way to test whether the problem is your firewall is to go to a WiFi location not behind your firewall. such as a coffee shop, and use Teams to send a request to your contact to chat. If the message goes through at the WiFi location, but does not when you're at work, then you know the problem is your firewall.
+To learn more about the ways that Teams users and Skype users can communicate, including limitations that apply, see [Teams and Skype interoperability](teams-skype-interop.md).
 
-> [!NOTE]
-> If you and another user both turn on external access and allow one another's domains, this will work. If it doesn't work, the other user should make sure their configuration isn't blocking your domain.
 
 ## Limit external access to specific people
 
-If you've enabled **People in my organization can communicate with Teams users whose accounts aren't managed by an organization**, you can limit external access to specific people by using PowerShell.
+If you've enabled any of the external access controls at a tenant-level, you can limit external access to specific users using PowerShell. People
```
[the remark this answers is not in the document]
Comment (JohanFreelancer9) [reply Teams/manage-external-access.md:153]: @ASolom1, I am not sure about this - should it be "PowerShell. People" (with a space)?

File: Teams/manage-external-access.md [hunk 8]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
-> [!NOTE]
-> If you and another user both turn on external access and allow one another's domains, this will work. If it doesn't work, the other user should make sure their configuration isn't blocking your domain.
 
 ## Limit external access to specific people
 
-If you've enabled **People in my organization can communicate with Teams users whose accounts aren't managed by an organization**, you can limit external access to specific people by using PowerShell.
+If you've enabled any of the external access controls at a tenant-level, you can limit external access to specific users using PowerShell. People 
 
-You can use the example script below, substituting *PolicyName* for the name you want to give the policy, and *UserName* for each user you want to be able to use external access.
+You can use the example script below, substituting *Control* for the control you want to change, *PolicyName* for the name you want to give the policy, and *UserName* for each user for which you want to enable/disablebe able to  use external access.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:155]:
```suggestion
You can use the following example script, substituting *Control* for the control you want to change, *PolicyName* for the name you want to give the policy, and *UserName* for each user for whom you want to enable/disable external access.
```

File: Teams/manage-external-access.md [hunk 9]
```diff
@@ -204,6 +214,18 @@ Incoming chats and calls from a federation organization will land in the user’
...
+|Pure on-premises | Chat with unmanaged Teams users is not supported for pure on-premises organizations.|
+|Hybrid with some users online (in either Skype for Business or Teams) and some users on-premises. | Follow above steps for online organizations. Please note that chat with unmanaged Teams users is not supported for on-premises users. |
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:225]:
```suggestion
|Hybrid with some users online (in either Skype for Business or Teams) and some users on-premises. | Follow the previously described steps for online organizations. Note that chat with unmanaged Teams users is not supported for on-premises users. |
```

File: Teams/manage-external-access.md [hunk 10]
```diff
@@ -219,6 +241,21 @@ To enable federation between users in your organization and consumer users of Sk
...
+
+To test your setup, you need a Teams user who's not behind your firewall.
+  
+1. After you and the admin from the organization have changed the **External access** settings, you should be good to go.
+
+2. In the Teams app, search for the person by email address, and send a request to chat.
+
+3. Ask your Teams contact to send you a request to chat. If you don't receive their request, the problem is your firewall settings (assuming they've already confirmed their firewall settings are correct).
+
+4. Another way to test whether the problem is your firewall is to go to a WiFi location not behind your firewall. such as a coffee shop, and use Teams to send a request to your contact to chat. If the message goes through at the WiFi location, but does not when you're at work, then you know the problem is your firewall.
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:254]:
```suggestion
4. Another way to test whether the problem is your firewall is to go to a Wi-Fi location that is not behind your firewall, such as a coffee shop, and use Teams to send a request to your contact to chat. If the message goes through at the Wi-Fi location but does not when you're at work, then you know the problem is your firewall.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ASolom1, kindly commit the suggested changes if you agree with them. Many thanks.

File: Teams/manage-external-access.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (ASolom1) [reply Teams/manage-external-access.md:153]: I think that we don't need the word "People" and that was a miss on my part. I updated "people" to "users" earlier in the sentence. Can you review with that context and see if it makes sense?

**Comment by ASolom1** [comment]
@JohanFreelancer9 - committed most changes and added a reply to one of your comments. Are you also able to check that formatting looks ok? I think I had some issues with lists.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@ASolom1 - I have changed the remaining suggestion to remove the word "People". Could you please commit the suggestion? As for the layout, it looks fine to me. Many thanks.

**Comment by ASolom1** [comment]
The bullet structure in "Manage contact with external Teams users not managed by an organization" seems off in the rich preview. The bullets don't seem tabbed properly. I'm happy to suggest a change here, but do not know how to do multi-layer lists in GitHub.

File: Teams/manage-external-access.md [hunk 11]
```diff
@@ -92,75 +111,67 @@ To block specific domains
...
+               2.	Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
+      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
+               1.	Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
+               2.	Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
+      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>
+               1.	Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting.<br>
+               2.	Select the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:134]:
```suggestion
1. In the left navigation, go to **Users** > **External access**.<br>
2. Follow one of these steps:<be>
    - To block Teams users in your organization from communicating with external Teams users whose accounts are not managed by an organization:<be>
    	1) Turn off the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
        2) Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
    - To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
        1) Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
        2) Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
    - To let Teams users In your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>
        1) Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting.<br>
        2) Select the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@ASolom1 - unfortunately, I cannot look at the preview to see what my suggestions look like. Please have a look and see if it works.

**Comment by ASolom1** [comment]
Looks much better, thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@ASolom1 please contact me internally to discuss this update.

**Comment by MikePlumleyMSFT** [comment]
Publishing this update through a different channel.

**Comment by MikePlumleyMSFT** [comment]
Publishing this update through a different channel.

**Pull request closed without merging**